src/functions/accent.ts

```diff
@@ -10,8 +9,8 @@ import {assertSpan, assertSymbolDomNode} from "../domTree";
 import * as html from "../buildHTML";
 import * as mml from "../buildMathML";
 
-import type {ParseNode, AnyParseNode} from "../parseNode";
-import type {HtmlBuilderSupSub, MathMLBuilder} from "../defineFunction";
```

Comment:
I heard that TypeScript is going to be introducing explicit type imports.  I think dropping them is fine though since TS is pretty good about backwards compatibility.